src/pages/profile/profile.js

```diff
@@ -1,127 +1,66 @@
 import "./profile.styles.css";
-import { Header, Footer } from "components";
-import { useState } from "react";
-import { Link, NavLink } from "react-router-dom";
-
-function getUserInfo() {
-  return JSON.parse(localStorage.getItem("userInfo"));
-}
+import { Loader } from "components";
+import { useAuthContext } from "context";
+import { useState, useEffect } from "react";
 
 function Profile() {
-  const [isNavFullLength, setIsNavFullLength] = useState(false);
-  const userInfo = getUserInfo();
+  const { authState } = useAuthContext();
+  const [userDetails, setUserDetails] = useState(false);
+  const [loader, setLoader] = useState("flex");
 
   //@TODO: This says I dont understand "this"
   function getItem(item) {
-    return userInfo[item] ?? "- not added -";
+    return userDetails[item] ?? "- not added -";
   }
 
+  function getuserInfo() {
+    const userInfo = JSON.parse(localStorage.getItem("userInfo"));
+    return userInfo;
```

Comment:
you may directly return `JSON.parse(localStorage.getItem("userInfo"))` instead of using a variable

Reply:
Done